server/server.go

```diff
@@ -510,7 +505,10 @@ func New(config *Config) (*Server, error) {
 		return nil, err
 	}
 
-	containerServer := libkpod.New(r, store, imageService, config.SignaturePolicyPath)
+	containerServer, err := libkpod.New(libkpod.DefaultConfig())
```

Comment:
Why default config here? Why not configure it from the config we're getting from file/command line?